src/main/java/ui/issuepanel/FilterPanel.java

```diff
@@ -230,6 +231,16 @@ private void applyFilterExpression(FilterExpression filter) {
 
     protected abstract void stopLoadingAnimationIfApplicable(AppliedFilterEvent e);
 
+    private void showErrorStatus(FilterExceptionEvent e) {
+        if (e.panel != this) return;
```

Comment:
It throws a ParseException on malformed input (and is what is causing our `is:` woes :smiling_imp:).